README.md

```diff
@@ -3,6 +3,7 @@
 [![Build Status](https://travis-ci.org/prasadtalasila/IRCLogParser.svg?branch=dev)](https://travis-ci.org/prasadtalasila/IRCLogParser) 
 [![Maintainability](https://api.codeclimate.com/v1/badges/211e8675682e2d345d8b/maintainability)](https://codeclimate.com/github/prasadtalasila/IRCLogParser/maintainability)
 [![Test Coverage](https://api.codeclimate.com/v1/badges/211e8675682e2d345d8b/test_coverage)](https://codeclimate.com/github/prasadtalasila/IRCLogParser/test_coverage) 
```

Comment:
In the README.md that is rendered in the browser, we can't distinguish between the coverage reported by codeclimate and coveralls. Also it is better to use the unit test coverage badge from codecov.

It is better to say, Unit Tests - [![codecov](https://codecov.io/gh/prasadtalasila/IRCLogParser/branch/dev/graph/badge.svg)](https://codecov.io/gh/prasadtalasila/IRCLogParser), Integration Tests - [![Test Coverage](https://api.codeclimate.com/v1/badges/211e8675682e2d345d8b/test_coverage)](https://codeclimate.com/github/prasadtalasila/IRCLogParser/test_coverage), Functional Tests - [![test coverage](https://coveralls.io/repos/github/prasadtalasila/IRCLogParser/badge.svg?branch=dev)](https://coveralls.io/github/prasadtalasila/IRCLogParser?branch=dev)

codecov - `[![codecov](https://codecov.io/gh/prasadtalasila/IRCLogParser/branch/dev/graph/badge.svg)](https://codecov.io/gh/prasadtalasila/IRCLogParser)`    
coveralls - `[![test coverage](https://coveralls.io/repos/github/prasadtalasila/IRCLogParser/badge.svg?branch=dev)]`    
codeclimate - `[![Test Coverage](https://api.codeclimate.com/v1/badges/211e8675682e2d345d8b/test_coverage)](https://codeclimate.com/github/prasadtalasila/IRCLogParser/test_coverage)`